test/integration/02-user-wallet/02-send-lightning.spec.ts

```diff
@@ -372,16 +420,29 @@ describe("UserWallet - Lightning Pay", () => {
           expect(payeeFinalBalance).toBe(payeeInitialBalance + amountInvoice)
 
           const hash = getHash(request)
-          const matchTx = (tx) => tx.type === "on_us" && tx.hash === hash
-
-          const user2Txn = await walletPayee.getTransactions()
+          const matchTx = (tx) =>
+            tx.settlementVia === "intraledger" && tx.paymentHash === hash
+
+          let txResult = await Wallets.getTransactionsForWalletId({
+            walletId: walletPayee.user.id,
+          })
+          if (txResult.error instanceof Error) {
+            throw txResult.error
+          }
+          const user2Txn = txResult.transactions
           const user2OnUsTxn = user2Txn.filter(matchTx)
-          expect(user2OnUsTxn[0].type).toBe("on_us")
```

Comment:
In places like this, do we also want to check for something like `expect(user2OnUsTxn[0].old.type).toBe("on_us")` since we're doing it in other places?

Reply:
Should probably only check for the consistently `settlementVia` but don't want to clean it all up now.